server/src/main/resources/application.properties

```diff
@@ -1,3 +1,5 @@
+application.version=@project.version@
+application.title=@project.name@
```

Comment:
```suggestion
```

Since in the server the variables are properly loaded, we don't need to write them additionally into the `properties` file.